nixos/configuration.nix

```diff
@@ -100,6 +100,8 @@
 
   services.cloudflare-warp = {
     enable = true;
+    # Using latest to avoid GH-749
+    package = edge-pkgs.cloudflare-warp;
```

Comment:
Not yet updated https://github.com/NixOS/nixpkgs/blob/cc90561a8fb233a54460b095110605171ea6578a/pkgs/tools/networking/cloudflare-warp/default.nix#L17

https://developers.cloudflare.com/warp-client/get-started/linux/
https://developers.cloudflare.com/cloudflare-one/changelog/warp/ - Looks like released newer

Reply:
Logs via WSL2(Ubuntu 24.04)

```
> sudo apt-get install cloudflare-warp
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  alsa-topology-conf alsa-ucm-conf cpp cpp-13 cpp-13-x86-64-linux-gnu cpp-x86-64-linux-gnu gcc-13-base libasound2-data libasound2t64 libauthen-sasl-perl libclone-perl
  libdata-dump-perl libencode-locale-perl libfile-basedir-perl libfile-desktopentry-perl libfile-listing-perl libfile-mimeinfo-perl libfont-afm-perl libgles2
  libhtml-form-perl libhtml-format-perl libhtml-parser-perl libhtml-tagset-perl libhtml-tree-perl libhttp-cookies-perl libhttp-daemon-perl libhttp-date-perl
  libhttp-message-perl libhttp-negotiate-perl libice6 libio-html-perl libio-socket-ssl-perl libio-stringy-perl libipc-system-simple-perl libisl23 liblwp-mediatypes-perl
  liblwp-protocol-https-perl libmailtools-perl libmpc3 libnet-dbus-perl libnet-http-perl libnet-smtp-ssl-perl libnet-ssleay-perl libsm6 libtie-ixhash-perl
  libtimedate-perl libtry-tiny-perl libu2f-udev liburi-perl libwww-perl libwww-robotrules-perl libx11-protocol-perl libxaw7 libxcb-shape0 libxft2 libxkbfile1
  libxml-parser-perl libxml-twig-perl libxml-xpathengine-perl libxmu6 libxpm4 libxt6t64 libxv1 libxxf86dga1 perl-openssl-defaults x11-utils x11-xserver-utils xdg-utils
  zutty
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  desktop-file-utils gnupg2 ibverbs-providers libibverbs1 libnftables1 libnftnl11 libnl-3-200 libnl-route-3-200 libnss3-tools libpcap0.8t64 nftables
Suggested packages:
  traceroute firewalld
The following NEW packages will be installed:
  cloudflare-warp desktop-file-utils gnupg2 ibverbs-providers libibverbs1 libnftables1 libnftnl11 libnl-3-200 libnl-route-3-200 libnss3-tools libpcap0.8t64 nftables
0 upgraded, 12 newly installed, 0 to remove and 0 not upgraded.
Need to get 40.5 MB of archives.
After this operation, 113 MB of additional disk space will be used.
Do you want to continue? [Y/n] Y
Get:1 http://archive.ubuntu.com/ubuntu noble/main amd64 libnl-3-200 amd64 3.7.0-0.3build1 [55.6 kB]
Get:2 https://pkg.cloudflareclient.com noble/main amd64 cloudflare-warp amd64 2024.6.497-1 [38.5 MB]
Get:3 http://archive.ubuntu.com/ubuntu noble/main amd64 libnl-route-3-200 amd64 3.7.0-0.3build1 [189 kB]
Get:4 http://archive.ubuntu.com/ubuntu noble/main amd64 libibverbs1 amd64 50.0-2build2 [67.8 kB]
Get:5 http://archive.ubuntu.com/ubuntu noble/main amd64 ibverbs-providers amd64 50.0-2build2 [374 kB]
Get:6 http://archive.ubuntu.com/ubuntu noble/main amd64 libnftnl11 amd64 1.2.6-2build1 [66.0 kB]
Get:7 http://archive.ubuntu.com/ubuntu noble/main amd64 libnftables1 amd64 1.0.9-1build1 [358 kB]
Get:8 http://archive.ubuntu.com/ubuntu noble/main amd64 libpcap0.8t64 amd64 1.10.4-4.1ubuntu3 [151 kB]
Get:9 http://archive.ubuntu.com/ubuntu noble/main amd64 nftables amd64 1.0.9-1build1 [69.8 kB]
Get:10 http://archive.ubuntu.com/ubuntu noble/universe amd64 gnupg2 all 2.4.4-2ubuntu17 [4748 B]
Get:11 http://archive.ubuntu.com/ubuntu noble/main amd64 desktop-file-utils amd64 0.27-2build1 [53.8 kB]
Get:12 http://archive.ubuntu.com/ubuntu noble/main amd64 libnss3-tools amd64 2:3.98-1build1 [615 kB]
Fetched 40.5 MB in 5s (8721 kB/s)
Selecting previously unselected package libnl-3-200:amd64.
(Reading database ... 42766 files and directories currently installed.)
Preparing to unpack .../00-libnl-3-200_3.7.0-0.3build1_amd64.deb ...
Unpacking libnl-3-200:amd64 (3.7.0-0.3build1) ...
Selecting previously unselected package libnl-route-3-200:amd64.
Preparing to unpack .../01-libnl-route-3-200_3.7.0-0.3build1_amd64.deb ...
Unpacking libnl-route-3-200:amd64 (3.7.0-0.3build1) ...
Selecting previously unselected package libibverbs1:amd64.
Preparing to unpack .../02-libibverbs1_50.0-2build2_amd64.deb ...
Unpacking libibverbs1:amd64 (50.0-2build2) ...
Selecting previously unselected package ibverbs-providers:amd64.
Preparing to unpack .../03-ibverbs-providers_50.0-2build2_amd64.deb ...
Unpacking ibverbs-providers:amd64 (50.0-2build2) ...
Selecting previously unselected package libnftnl11:amd64.
Preparing to unpack .../04-libnftnl11_1.2.6-2build1_amd64.deb ...
Unpacking libnftnl11:amd64 (1.2.6-2build1) ...
Selecting previously unselected package libnftables1:amd64.
Preparing to unpack .../05-libnftables1_1.0.9-1build1_amd64.deb ...
Unpacking libnftables1:amd64 (1.0.9-1build1) ...
Selecting previously unselected package libpcap0.8t64:amd64.
Preparing to unpack .../06-libpcap0.8t64_1.10.4-4.1ubuntu3_amd64.deb ...
Unpacking libpcap0.8t64:amd64 (1.10.4-4.1ubuntu3) ...
Selecting previously unselected package nftables.
Preparing to unpack .../07-nftables_1.0.9-1build1_amd64.deb ...
Unpacking nftables (1.0.9-1build1) ...
Selecting previously unselected package gnupg2.
Preparing to unpack .../08-gnupg2_2.4.4-2ubuntu17_all.deb ...
Unpacking gnupg2 (2.4.4-2ubuntu17) ...
Selecting previously unselected package desktop-file-utils.
Preparing to unpack .../09-desktop-file-utils_0.27-2build1_amd64.deb ...
Unpacking desktop-file-utils (0.27-2build1) ...
Selecting previously unselected package libnss3-tools.
Preparing to unpack .../10-libnss3-tools_2%3a3.98-1build1_amd64.deb ...
Unpacking libnss3-tools (2:3.98-1build1) ...
Selecting previously unselected package cloudflare-warp.
Preparing to unpack .../11-cloudflare-warp_2024.6.497-1_amd64.deb ...
Unpacking cloudflare-warp (2024.6.497-1) ...
Setting up gnupg2 (2.4.4-2ubuntu17) ...
Setting up desktop-file-utils (0.27-2build1) ...
Setting up libnftnl11:amd64 (1.2.6-2build1) ...
Setting up libnl-3-200:amd64 (3.7.0-0.3build1) ...
Setting up libnss3-tools (2:3.98-1build1) ...
Setting up libnftables1:amd64 (1.0.9-1build1) ...
Setting up nftables (1.0.9-1build1) ...
Setting up libnl-route-3-200:amd64 (3.7.0-0.3build1) ...
Setting up libibverbs1:amd64 (50.0-2build2) ...
Setting up ibverbs-providers:amd64 (50.0-2build2) ...
Setting up libpcap0.8t64:amd64 (1.10.4-4.1ubuntu3) ...
Setting up cloudflare-warp (2024.6.497-1) ...
-sh: 41: /home/kachick/.nix-profile/etc/profile.d/hm-session-vars.sh: [[: not found
-sh: 7: [[: not found
-sh: 41: /home/kachick/.nix-profile/etc/profile.d/hm-session-vars.sh: [[: not found
-sh: 7: [[: not found
Created symlink /etc/systemd/system/multi-user.target.wants/warp-svc.service → /usr/lib/systemd/system/warp-svc.service.
Processing triggers for libc-bin (2.39-0ubuntu8.2) ...
Processing triggers for man-db (2.12.0-4build2) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
```

Reply:
Looks like I can access to 

https://pkg.cloudflareclient.com/pool/noble/main/c/cloudflare-warp/cloudflare-warp_2024.6.497-1_amd64.deb